packages/gatsby-theme-docs/src/layouts/internals/page-navigation.js

```diff
@@ -35,10 +35,17 @@ function flatten(items) {
 
 const getIsActive = (activeSection, item, recursive = false) => {
   if (!activeSection) return false;
-  const itemAndChildUrls =
-    item.items && recursive
-      ? [item.url, ...flatten(item.items).map((flatChild) => flatChild.url)]
-      : [item.url];
+
+  let itemAndChildUrls;
+
+  if (item.items && recursive) {
+    const flattenedItems = flatten(item.items);
+    const flattenedItemUrls = flattenedItems.map((flatChild) => flatChild.url);
+    itemAndChildUrls = [item.url, ...flattenedItemUrls];
+  } else {
+    itemAndChildUrls = [item.url];
+  }
+
```

Comment:
just to be sure: beyond rewriting it in a less compressed way, does this represent a change?  I can't see one.

Reply:
No functionality change, this is just rewriting to make it easily readable.